packages/react/src/components/UIShell/components/styles/_user-info.scss

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Should we rename this to _profile.scss? and do we want to add a cds--profile class to the container component?